generator/generator_config.textproto

```diff
@@ -521,7 +521,6 @@ service {
   omit_stub_factory: true
   omitted_rpcs: [
     "DeleteBucket",
-    "GetBucket",
```

Comment:
only 20 RPCs left ☠️

Reply:
Yeah, and then about 30 calls that will not be implemented natively in gRPC and I will have to emulate. Happy joy joy.